src/devices/Pn532/Pn532.cs

```diff
@@ -2058,8 +2076,8 @@ private bool IsReady()
             if (_spiDevice is object && _controller is object)
             {
                 _controller.Write(_pin, PinValue.Low);
-                _spiDevice.WriteByte(ReadStatus);
-                ret = _spiDevice.ReadByte();
+                _spiDevice.WriteByte(ReverseByte(ReadStatus));
```

Comment:
if you always need to reverse byte I'd recommend to enforce DataFlow.LsbFirst instead in the constructor

Reply:
The thing is that Mod0 and Lbs first is not working on Raspberry Pi. So if this mode is reinforced, it will just crash. So I disregard the value and prefer people not to worry about it

Reply:
:( we should fix that, if not supported by driver then do it with software

Reply:
(out of scope for this PR)

Reply:
https://github.com/dotnet/iot/issues/1308